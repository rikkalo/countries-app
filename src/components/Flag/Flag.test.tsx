import { shallow } from "enzyme";
import React from "react";

import { Flag } from "./index";

describe("Flag", () => {
  it("should render correctly", () => {
    const wrapper = shallow(
      <Flag value="https://restcountries.eu/data/asm.svg" />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
