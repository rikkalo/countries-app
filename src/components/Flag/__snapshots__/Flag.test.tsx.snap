// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flag should render correctly 1`] = `
<img
  alt="Flag"
  className="card"
  src="https://restcountries.eu/data/asm.svg"
/>
`;
